addon/components/smart-link-to/template.hbs

```diff
@@ -0,0 +1,15 @@
+{{#if isRoute}}
+  {{#if target}}
+    {{#link-to link class=cssClass target=target}}
+      {{yield}}{{{label}}}
+    {{/link-to}}
+  {{else}}
+    {{#link-to link class=cssClass}}
+      {{yield}}{{{label}}}
+    {{/link-to}}
+  {{/if}}
```

Comment:
By default `target` value of a `link-to` is nul, no ? If it's the case the `if/else` is maybe useless no ?